src/frontend/src/lib/components/transactions/Transaction.svelte

```diff
@@ -24,7 +24,7 @@
 	$: icon = type === 'send' ? IconSend : IconReceive;
 </script>
 
-<button class="contents">
+<button class="contents" on:click>
```

Comment:
It works with the class `contents`?

In `IcTransaction`n the style is `class="block w-full border-0"`

Reply:
Yes, it works with just `contents` - in prod, it's already used for ETH transactions. Also, I tried it for IC locally - also looking good. Here is a screenshot of a BTC tx:

<img width="1545" alt="Screenshot 2024-10-02 at 08 59 26" src="https://github.com/user-attachments/assets/36e64d45-6c58-4c0a-a57e-959862a608b0">

I will still triple-check it when creating a PR for using `Transaction.svelte` by all networks just to be on the safe side.